actions/raw_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+Raw Action
+
+Allows to write some file to output image with given offset.
```

Comment:
Directly write a file to the output image at a given offset

Reply:
done